docs/data-sources/cluster.md

```diff
@@ -55,7 +55,7 @@ The following arguments are supported:
 - `config_store` - (Computed) - [config_store_spec](#config_store_spec) - ConfigStore configures the stores that nodes use to get their configuration.
 - `cloud_provider` - (Computed) - [cloud_provider_spec](#cloud_provider_spec) - CloudProvider configures the cloud provider to use.
 - `gossip_config` - (Computed) - [gossip_config](#gossip_config) - GossipConfig for the cluster assuming the use of gossip DNS.
-- `container_runtime` - (Computed) - String - Container runtime to use for Kubernetes.
+- `container_runtime` - (Computed) - String - ContainerRuntime was removed.
```

Comment:
Ah yes it comes from the kops code: https://github.com/kubernetes/kops/pull/15871/files

It's a shame this provider isn't on the newere TF plugin framework, otherwise we could potentially added a `DeprecationMessage` for these fields.